src/CMakeLists.txt

```diff
@@ -123,11 +135,13 @@ target_include_directories(dispatch
                              ${CMAKE_CURRENT_SOURCE_DIR}
                              ${CMAKE_CURRENT_BINARY_DIR}
                              ${CMAKE_SOURCE_DIR}/private)
+
```

Comment:
I'd recommend against adding blank lines as it adds more noise to the patch than is necessary to get your changes through